docs/source/upgrade_to_newest_version.md

```diff
@@ -0,0 +1,114 @@
+# Considerations for getting to v2.0
+
+## Chaincode lifecycle
+
+The new chaincode lifecycle that debuts in v2.0 allows multiple organizations to agree on how a chaincode will be operated before it can be used on a channel. For more information about the new chaincode lifecycle, check out [Chaincode for operators](./chaincode4noah.html).
+
+It is a best practice to upgrade all of the peers on a channel before enabling the `Channel` and `Application` capabilities that enable the new chaincode lifecycle. Note that any peers that are not at v2.0 will crash after enabling either capability, while any ordering nodes that are not at v2.0 will crash after the `Channel` capability has been enabled. This crashing behavior is intentional, as the peer or orderer cannot safely participate in the channel if it does not support the required capabilities.
```

Comment:
Channel capability is not required to enable new chaincode lifecycle, just Application capability.
But it does make sense to update them at the same time.  I have a comment later on mentioning that Channel capability V2_0 is actually a noop.